astropy/modeling/polynomial.py

```diff
@@ -141,6 +141,10 @@ class OrthoPolynomialBase(PolynomialBase):
 
     The polynomials implemented here require a maximum degree in x and y.
 
+    For explanation of `x_domain`, `y_domain`, `x_window` and `y_window`
```

Comment:
parameter names, here and below, need to be double backticked, otherwise it'll cause the sphinx build fails.

Reply:
Thanks @bsipocz, all fixed now.